packages/lesswrong/server/vulcan-lib/mutators.ts

```diff
@@ -477,7 +472,7 @@ export const updateMutator: UpdateMutator = async <N extends CollectionNameStrin
     properties,
   ]);
   
-  void logFieldChanges({currentUser, collection, oldDocument, data: origData});
+  void require('../fieldChanges').logFieldChanges({currentUser, collection, oldDocument, data: origData});
```

Comment:
I gotta say I don't love dynamic requires, but don't see how to avoid this one without literally just moving `logFieldChanges` into this file or something.